.github/workflows/test.yaml

```diff
@@ -34,10 +33,6 @@ jobs:
       run: sudo apt build-dep -y ${{ github.workspace }}/src/github.com/snapcore/snapd
     - name: Make /var/cache/apt owned by current user
       run: sudo chown -R $(id -u) /var/cache/apt
-    - name: Set up go 1.9
```

Comment:
Why is this removed? Is it no longer needed?

Reply:
Because we get both debian and custom go and I got an error where go said object files were built with different compiler.

Reply:
So we rely on running the unit tests in the spread runs?

Reply:
No, we run unit tests here, look below.

Reply:
Sorry if I sound dumb - but how do we know that our stuff still works with go-1.9 if we don't ask for 1.9 in the unit tests check?

Reply:
We don't, I'm trying to get the best of what's available here.

I'll update this as I'm progressing. I think the tests will actually pass now.

Reply:
Ok, we discussed this some more and we're now using go 1.10 from the xenial archive. In a follow-up we will switch to go 1.9 from a snap.